http/http_client.hpp

```diff
@@ -783,15 +784,16 @@ class ConnectionPool : public std::enable_shared_from_this<ConnectionPool>
         }
         else if (requestQueue.size() < maxRequestQueueSize)
         {
-            BMCWEB_LOG_ERROR << "Max pool size reached. Adding data to queue."
-                             << destIP << ":" << std::to_string(destPort);
+            BMCWEB_LOG_CRITICAL
```

Comment:
Lets move this to DEBUG ( as done in 1050) or remove this change if you are getting that other PR merged to 1060 earlier than this.

Reply:
done